label_sync/labels.yaml

```diff
@@ -14,16 +14,6 @@ default:
       target: prs
       prowPlugin: approve
       addedBy: approvers
-    - color: fef2c0
-      description: Indicates a cherrypick PR into a release branch has been approved by the release branch manager. # Consumed by the kubernetes/kubernetes cherrypick-queue.
-      name: cherrypick-approved
```

Comment:
Perhaps we should delete it from all the other repos by a) giving it a deleteAfter field? Probably need to teach the tool how to handle a deleted org label with a present repo label